.github/workflows/sdk-test-suite.yaml

```diff
@@ -12,10 +12,18 @@ on:
         required: false
         description: Set the e2e-api tests branch tag
         default: master
-      react_version:
+      react_version: # deprecated
         type: string
         required: false
         description: Set the react version
+      client_framework:
```

Comment:
I think this can be a pre-defined list of values, so users of this workflow will not have to type in manually (typos might cause issues)

Reply:
no one should type it. it's an internal file. who will trigger it will have a dropdown in your e2e-repo

Reply:
(Unless this workflow cannot be triggered manually)